composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "license": "MIT",
     "require": {
-        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*"
+        "laravel/framework": "5.0.*|5.1.*|5.2.*|5.3.*|5.4.*|5.5.*"
```

Comment:
@mattstauffer I'd actually love your opinion on that given the circumstances. We can't fully be sure until Laravel 5.5 is officially released (we could test off the latest dev version of Laravel of course, but there could be issues that come from it being dev). Given that package auto-discovery doesn't exist before 5.5, should I just re-PR this once Laravel 5.5 releases and we can fully test it on a stable release? Or is there any value in doing testing right now while 5.5 is still in dev?